doc/source/whatsnew/v0.18.2.txt

```diff
@@ -528,3 +528,5 @@ Bug Fixes
 
 - Bug in ``Categorical.remove_unused_categories()`` changes ``.codes`` dtype to platform int (:issue:`13261`)
 - Bug in ``groupby`` with ``as_index=False`` returns all NaN's when grouping on multiple columns including a categorical one (:issue:`13204`)
+
+- Bug in ``pandas.formats.format.Datetime64Formatter`` was not respecting a formatter function passed in as an argument (:issue:`10690`)
```

Comment:
a user is reading this, say something like: ``date_format`` kw was being ignored when passed a callable and ``.to_string`` or ``.to_html``